src/components/Home/Home.jsx

```diff
@@ -23,6 +24,7 @@ const Home = () => {
   return (
     <>
       <header>
+
         <div className="top-bar">
           <img src={home} alt="Home Icon" className="home-img" />
           <h1>Countries</h1>
```

Comment:
### Design Head
![image](https://user-images.githubusercontent.com/93039359/236442262-f7476efb-0880-4f26-9c4b-54b794bac0c4.png)

### Your Head
![image](https://user-images.githubusercontent.com/93039359/236442591-d211027b-52d7-43ba-81b3-bbd14ee1fb34.png)

- [ ] As you can see in the image above that your heading and image should be on right and left but in your design i can only see your image because your heading is at the top 😨 Kindly fix this to make your project look like given design to follow the project requirements 👍